vendor/deps/cli-kit/lib/cli/kit/system.rb

```diff
@@ -227,7 +227,7 @@ def resolve_path(a, env)
         end
 
         def which(cmd, env)
-          exts = os == :windows ? env.fetch('PATHEXT', ['']).split(';') : ['']
+          exts = (os == :windows && env['PATHEXT']&.split(';')) || ['']
```

Comment:
Alternatively:

```suggestion
          exts = 
            if os == :windows
              env['PATHEXT']&.split(';') || ['']
            else
              ['']
            end
```

Thoughts?